astropy/coordinates/builtin_frames/heliocentric.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from ..representation import SphericalRepresentation
+from ..baseframe import (BaseCoordinateFrame, RepresentationMapping,
+                         TimeFrameAttribute)
+from .utils import DEFAULT_OBSTIME
+
+class Heliocentric(BaseCoordinateFrame):
```

Comment:
If you explained the name in the docstring, and it's not used anywhere else, one would assume the astropy docs would come up at the top?!

You have made me think about the naming of SunPy's frames because the axes are not really in the names, especially for our heliocentric frame.